static/scripts/collapse.js

```diff
@@ -1,16 +1,14 @@
 function hideAllButCurrent(){
-    //by default all submenut items are hidden
```

Comment:
Like the comment says, they're already hidden by the template, so there's no reason to do this.